spec/requests/topics_controller_spec.rb

```diff
@@ -4061,6 +4061,84 @@ def extract_post_stream
         end
       end
     end
+
+    describe "new and unread" do
```

Comment:
Can we add tests cases for when the dismissal is scoped to a category, tag or specific tag IDs?